knowledge_graph/routes.py

```diff
@@ -152,6 +168,29 @@ def receive_user_scores() -> Tuple[Response, int]:
 
     response = Response(dumps(value_scores))
     return response, 200
+    
+
+@app.route('/personal_values', methods=['GET'])
```

Comment:
Is it more readable to have the session-id parameter as a query string here?

Reply:
What do you mean by this? Can you type some example code here?

Reply:
Sorry, ignore me! I had the syntax mixed up with something else, I thought you could put some parameter in the route name and have flask read it like 'personal_values?session-id=<parameter>', but this is right